bandwidth.gemspec

```diff
@@ -8,13 +8,17 @@ Gem::Specification.new do |s|
   s.homepage = 'https://github.com/Bandwidth/ruby-sdk'
   s.license = 'MIT'
   s.add_dependency('logging', '~> 2.3')
-  s.add_dependency('faraday', '~> 1.0', '<= 1.9.3')
-  s.add_dependency('faraday_middleware', '~> 1.0')
+  s.add_dependency('faraday', '>= 1.0', '< 3.0')
+  s.add_dependency('faraday-follow_redirects', '~> 0.3')
+  s.add_dependency('faraday-gzip', '~> 0.1.0')
+  s.add_dependency('faraday-multipart', '~> 1.0')
+  s.add_dependency('faraday-retry', '~> 1.0')
   s.add_dependency('certifi', '~> 2018.1', '>= 2018.01.18')
   s.add_dependency('faraday-http-cache', '~> 2.2')
   s.add_dependency('builder', '~> 3.2.4')
   s.add_development_dependency('minitest', '~> 5.14', '>= 5.14.1')
   s.add_development_dependency('minitest-proveit', '~> 1.0')
+  s.add_development_dependency('test-unit')

```

Comment:
Is there a reason you added this dependency? It seems to be unused unless there's something I'm missing

Reply:
It is required here https://github.com/Bandwidth/ruby-sdk/blob/main/test/integration/test_integration.rb#L5